### DISCUSSION TITLE
Texstudio编译参考文献后不显示[J],[D],[C]等文献类型

### DISCUSSION THREAD

**Discussion by susufang** [sjtug/SJTUThesis#1080] [提问]
问题一：【急】Texstudio编译参考文献后不显示[J],[D],[C]等文献类型。
问题二：表头与表格中间空行太大，图注和图则没有这个问题。

**Comment by LogCreative** [comment]
问题一：.bib 文件中的键有没有正确地按照 `@journal`, `@phdthesis`, `@inproceedings` 的类型进行标注。
问题二：如果你认为空行比较大，考虑在导言区添加
```latex
\captionsetup{skip=4bp} % 原本为 6bp
```
